### PR TITLE
Robo hash avatar

### PR DESCRIPTION
**Pull request by victor-dantas**
Add roboHash to user avatar

### CODE REVIEW

File: test/unit/specs/components/user/__snapshots__/UserAvatar.spec.js.snap [hunk 1]
```diff
@@ -1,13 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`UserAvatar.vue renders correctly 1`] = `
-<bhavatar-stub size="normal">
-  <bhicon-stub value="account" size="small"></bhicon-stub>
-</bhavatar-stub>
-`;
+exports[`UserAvatar.vue renders correctly 1`] = `<bhavatar-stub size="normal" imagelink="https://robohash.org/undefined.png"></bhavatar-stub>`;
```
Comment (douglasppaz) [inline test/unit/specs/components/user/__snapshots__/UserAvatar.spec.js.snap]: Set profile.nickname in spec to not returns undefined.png

File: bh/documentation/src/avatar/Avatar.vue [hunk 2]
```diff
@@ -26,6 +32,8 @@ import AvatarExample from './examples/AvatarExample';
 import avatarExampleCode from '!!raw-loader!./examples/AvatarExample';
 import AvatarSize from './examples/AvatarSize';
 import avatarSizeCode from '!!raw-loader!./examples/AvatarSize';
+import AvatarExampleImage from './examples/AvatarExampleImage';
```
Comment (douglasppaz) [inline bh/documentation/src/avatar/Avatar.vue]: Follow the pattern for examples:

file: examples/ComponentName.vue
component name: ComponentNameExample
component import: ComponentNameExample
component code import: componentNameExampleCode

File: bh/src/assets/scss/bh.scss [hunk 3]
```diff
@@ -133,3 +133,15 @@ pre {
...
   }
 }
+
+img {
+  @each $size in $sizes {
+    $size-name: nth($size, 1);
+    $size-value: nth($size, 2);
+
+    &.#{$size-name} {
+      max-width: $size-value;
```
Comment (douglasppaz) [inline bh/src/assets/scss/bh.scss]: remove this max-width, just force height

File: bh/src/components/BhAvatar.vue [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
-  <button
-    :class="{
-      'bh-avatar': true,
-      [`bh-avatar--${size}`]: !!size,
-    }"
-    @click="$emit('click', $event)"
-  >
-    <div class="bh-avatar__content"><slot /></div>
-  </button>
+  <div>
```
Comment (douglasppaz) [inline bh/src/components/BhAvatar.vue]: remove this wrapper, call button directly.

File: test/unit/specs/components/user/__snapshots__/UserAvatar.spec.js.snap [hunk 5]
```diff
@@ -1,13 +1,5 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`UserAvatar.vue renders correctly 1`] = `
-<bhavatar-stub size="normal">
-  <bhicon-stub value="account" size="small"></bhicon-stub>
-</bhavatar-stub>
-`;
+exports[`UserAvatar.vue renders correctly 1`] = `<bhavatar-stub size="normal"><img src="https://robohash.org/undefined" class="small"></bhavatar-stub>`;
```
Comment (douglasppaz) [inline test/unit/specs/components/user/__snapshots__/UserAvatar.spec.js.snap]: in spec set props data with profile.nickname info

File: bh/documentation/src/avatar/Avatar.vue (hunk 2 above)
Comment (victor-dantas) [reply bh/documentation/src/avatar/Avatar.vue]: I got it

**Pull request merged**